invoke stubs report not callable

Example:
```python
from invoke import task, Context

@task
def foo(context: Context) -> None:
    pass

@task
def bar(context: Context) -> None:
    pass

@task(foo, bar)
def baz(context: Context) -> None:
    pass
```

Error: `foo.py:15:2: error: <nothing> not callable  [misc]`

I'm the author of the stubs, but I don't understand the error. Can someone please support?

--- a/stubs/invoke/invoke/tasks.pyi
+++ b/stubs/invoke/invoke/tasks.pyi
@@ -73,7 +73,7 @@ def task(
     iterable: Iterable[str] | None = ...,
     incrementable: Iterable[str] | None = ...,
     klass: type[_TaskT] = ...,
-) -> _TaskT: ...
+) -> Callable[[Callable[..., Any]], _TaskT]: ...
 
 class Call:
     task: Task

